rpc/RequestBuilder.java

```diff
@@ -182,7 +182,7 @@
             }
         }
 
-        public static Collection<ConceptProto.Concept> concepts(Collection<Concept<?>> concepts) {
```

Comment:
The reason this shaky code was "working" before was because we never passed it a List - only an Array, and arrays in Java are not type checked as strictly as generic collections.